l5r/models/techviewmodel.py

```diff
@@ -105,18 +105,13 @@ def clean(self):
     def update_from_model(self, model):
         self.clean()
 
-        #for r in api.character.rankadv.get_all():
-        #    tech_ = api.character.schools.get_tech_by_rank(r.rank)
-        #    if tech_:
-        #        self.add_item(tech_, r.rank)
-
-        for i in range(1, 10):
-            tech_ = api.character.schools.get_tech_by_rank(i)
+        for r in api.character.rankadv.get_all():
+            tech_ = api.character.schools.get_tech_by_rank(r.rank)
             if tech_:
-                self.add_item(tech_, i)
+                self.add_item(tech_, r.rank)
```

Comment:
About this, I'm not sure this can cover all the use cases. I've tried this approach before but then changed it in this commit:
https://github.com/OpenNingia/l5r-character-manager-3/commit/50ba0ba1bbbdb04f6deca9e848c5ea418627aace

My goal was to allow choosing a Path as the first rank and maybe I took the long way around. Can you check if this still works with your commit?

Reply:
No it doesn't you are right.
For example I tested with the Deathseeker path and it adds it twice to the techniques list.

I also need to raise a bug. I also tested with the Moshi Guardians of the Sun path, when you create a character the Moshi Guardians of the Sun path isn't available.